setup.py

```diff
@@ -24,6 +24,8 @@
         "torch>=1.3.0",
         "torch_complex",
         "nltk>=3.4.5",
+        # lhotse
+        "lhotse @ git+https://github.com/lhotse-speech/lhotse",
```

Comment:
The form  of "@git+" is not allowed in setup.py because it prevents uploading the module to pypi sever

Reply:
But then shouldn't the errors be something else ?
Like all errors right now are due to numpy in some way. 

Anyway, I will remove from setup.py for now and add it in egs2/chime7_task1/local/install_dependencies.sh
When they release a new version with the fixes we need I will add it to setup.py. 
It is bad practice anyway to install always the latest master version.

Reply:
I simply tell you the restriction about setup.py. This is not related to the error.
I don't know why the error happens. Maybe, lhotse requires the older numpy version.  Numpy was degraded in the CI test: 1.24 -> 1.22

https://github.com/espnet/espnet/actions/runs/4098431848/jobs/7067535502#step:5:60
https://github.com/espnet/espnet/actions/runs/4098431848/jobs/7067535502#step:5:2636

Reply:
Ok thank you very much ! Anyway I moved lhotse from setup.py until this issue with numpy is resolved.